dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1979,20 +1987,16 @@
       "type": "object",
       "description": "Supported parameters for an Extractive Summarization task.",
       "properties": {
-        "sentenceCount": {
-          "type": "integer",
-          "default": 3
-        },
         "sortBy": {
           "$ref": "#/definitions/ExtractiveSummarizationSortingCriteria"
-        },
-        "stringIndexType": {
-          "$ref": "common.json#/definitions/StringIndexType"
```

Comment:
Did these GA in 2022-05-01? If so, you can't change the - even if the resulting model would look the same in REST (it'll break SDKs).

Reply:
@heaths, thanks for letting me know that. I confirm the ExtractiveSummarization has never GAed before. Since ExtractiveSummarizationTaskParameters is always in preview, is it OK to move the properties to base class?

Reply:
I have reverted all the changes to Extractive Summarization, because sentenceCount default value is different between Extractive and Abstractive Summarization tasks.